src/main/java/fi/fmi/avi/converter/tac/lexer/impl/util/GeometryHelper.java

```diff
@@ -94,17 +95,50 @@ private static void addCoordinateSeparatorLexemes(final List<Lexeme> lexemes, fi
         lexemes.add(createLexeme.apply(MeteorologicalBulletinSpecialCharacter.SPACE.getContent(), LexemeIdentity.WHITE_SPACE));
     }
 
+    /**
+     * Creates lexemes for a geometry with a specific winding order enforced.
+     *
+     * @param geom          the geometry
+     * @param createLexeme  function to create lexeme instances
+     * @param specifyZeros  if true, always includes minutes even when zero
+     * @param decimalPlaces number of decimal places for coordinate rounding
+     * @param winding       the winding order to enforce
+     * @return list of lexemes representing the geometry
+     */
     public static List<Lexeme> getGeoLexemes(final Geometry geom,
                                              final BiFunction<String, LexemeIdentity, Lexeme> createLexeme,
-                                             final boolean specifyZeros) {
-        return getGeoLexemes(geom, createLexeme, specifyZeros, 2, Winding.CLOCKWISE);
+                                             final boolean specifyZeros,
+                                             final int decimalPlaces,
+                                             final Winding winding) {
+        return getGeoLexemesInternal(geom, createLexeme, specifyZeros, decimalPlaces, winding);
     }
 
+    /**
+     * Creates lexemes for a geometry, preserving the original winding order.
+     * <p>
+     * Use this method when the polygon coordinates should not be reordered,
+     * for example when the input data is already in the correct order and
+     * winding detection may be unreliable (e.g., SWX polygons crossing the antimeridian).
```

Comment:
Maybe replace the example of an unreliable case with a reference to `Winding`, with a note like
```
see limitations of {@link Winding}...
```
? To have a single source of truth for the limitations.